### ISSUE TITLE
Crash: incompatibility with ServerCore 

### ISSUE THREAD

**Issue by CapoFantasma97** [devs-immortal/The-Aether#562]
### What were you doing when the crash occured?

The game crashes on startup when ServerCore is installed.

According to the developer, the cause is:

> ServerCore contains a patch that stops maps from loading chunks (causing lagspikes), which can be seen [here](https://github.com/Wesley1808/ServerCore-Fabric/blob/1.18/src/main/java/org/provim/servercore/mixin/optimizations/chunk_loading/MapItemMixin.java).
> The Aether Reborn seems to [mixin override the entire method](https://github.com/devs-immortal/Paradise-Lost/blob/1.17/src/main/java/net/id/aether/mixin/item/FilledMapMixin.java), making basically any mod touching player held map functionality at all - incompatible, causing crashes.
> 
> This is not an issue I can easily fix on my end (without completely removing the patch).

The game did not generate any crash report, only logs.

### Did it crash the client or server?

Both

### Mod Version

1.6.3.1 (Latest)

### Fabric API Version

0.12.8

### Trinkets Version

3.0.4

### Crash Report

```shell
[17:35:26] [main/INFO]: Loading for game Minecraft 1.17.1
[17:35:26] [main/INFO]: Loading 73 mods:
	- architectury 2.6.35
	- cardinal-components-base 3.0.0 via the_aether
	- cardinal-components-entity 3.0.0 via the_aether
	- cloth-api 2.0.54
	- cloth-basic-math 0.6.0 via cloth-config2
	- cloth-client-events-v0 2.0.54 via cloth-api
	- cloth-common-events-v1 2.0.54 via cloth-api
	- cloth-config2 5.1.40
	- cloth-datagen-api-v1 2.0.54 via cloth-api
	- cloth-scissors-api-v1 2.0.54 via cloth-api
	- cloth-utils-v1 2.0.54 via cloth-api
	- com_electronwill_night-config_core 3.6.4 via servercore-fabric
	- com_electronwill_night-config_toml 3.6.4 via servercore-fabric
	- crowdin-translate 1.3+1.17 via the_aether
	- customportalapi 0.0.1-beta46-1.17 via the_aether
	- fabric 0.43.1+1.17
	- fabric-api-base 0.4.0+cf39a74318 via fabric
	- fabric-api-lookup-api-v1 1.3.2+cf39a74318 via fabric
	- fabric-biome-api-v1 3.2.2+cf39a74318 via fabric
	- fabric-blockrenderlayer-v1 1.1.6+cf39a74318 via fabric
	- fabric-command-api-v1 1.1.4+cf39a74318 via fabric
	- fabric-commands-v0 0.2.3+cf39a74318 via fabric
	- fabric-containers-v0 0.1.13+cf39a74318 via fabric
	- fabric-content-registries-v0 0.4.1+3447790d18 via fabric
	- fabric-crash-report-info-v1 0.1.6+cf39a74318 via fabric
	- fabric-dimensions-v1 2.0.13+cf39a74318 via fabric
	- fabric-entity-events-v1 1.4.1+377137cc18 via fabric
	- fabric-events-interaction-v0 0.4.11+cf39a74318 via fabric
	- fabric-events-lifecycle-v0 0.2.3+cf39a74318 via fabric
	- fabric-game-rule-api-v1 1.0.8+cf39a74318 via fabric
	- fabric-item-api-v1 1.2.5+cf39a74318 via fabric
	- fabric-item-groups-v0 0.3.2+cf39a74318 via fabric
	- fabric-key-binding-api-v1 1.0.6+2a2bb57318 via fabric
	- fabric-keybindings-v0 0.2.4+cf39a74318 via fabric
	- fabric-lifecycle-events-v1 1.4.6+0392f3a618 via fabric
	- fabric-loot-tables-v1 1.0.5+cf39a74318 via fabric
	- fabric-mining-level-api-v1 1.0.3+cf39a74318 via fabric
	- fabric-mining-levels-v0 0.1.6+cf39a74318 via fabric
	- fabric-models-v0 0.3.1+cf39a74318 via fabric
	- fabric-networking-api-v1 1.0.14+cf39a74318 via fabric
	- fabric-networking-blockentity-v0 0.2.12+cf39a74318 via fabric
	- fabric-networking-v0 0.3.3+cf39a74318 via fabric
	- fabric-object-builder-api-v1 1.10.12+cf39a74318 via fabric
	- fabric-object-builders-v0 0.7.6+cf39a74318 via fabric
	- fabric-particles-v1 0.2.5+cf39a74318 via fabric
	- fabric-registry-sync-v0 0.7.13+cf39a74318 via fabric
	- fabric-renderer-api-v1 0.4.5+cf39a74318 via fabric
	- fabric-renderer-indigo 0.4.9+cf39a74318 via fabric
	- fabric-renderer-registries-v1 3.2.6+cf39a74318 via fabric
	- fabric-rendering-data-attachment-v1 0.1.6+cf39a74318 via fabric
	- fabric-rendering-fluids-v1 0.2.1+cf39a74318 via fabric
	- fabric-rendering-v0 1.1.7+cf39a74318 via fabric
	- fabric-rendering-v1 1.10.1+377137cc18 via fabric
	- fabric-resource-loader-v0 0.4.10+f09604ce18 via fabric
	- fabric-screen-api-v1 1.0.5+cf39a74318 via fabric
	- fabric-screen-handler-api-v1 1.1.9+cf39a74318 via fabric
	- fabric-structure-api-v1 1.1.15+cf39a74318 via fabric
	- fabric-tag-extensions-v0 1.2.3+cf39a74318 via fabric
	- fabric-textures-v0 1.0.7+cf39a74318 via fabric
	- fabric-tool-attribute-api-v1 1.3.2+cf39a74318 via fabric
	- fabric-transfer-api-v1 1.5.2+cf39a74318 via fabric
	- fabricloader 0.12.8
	- gud_recipe_confidence 1.0.2 via the_aether
	- incubus_core 1.6.3 via the_aether
	- java 16
	- minecraft 1.17.1
	- moretags 1.2.5 via the_aether
	- notenoughcrashes 4.1.2+1.17.1
	- reach-entity-attributes 2.1.1 via the_aether
	- seedy-behavior 1.0.1 via the_aether
	- servercore-fabric 1.2.5-1.17.1
	- the_aether 1.17.1-1.6.3.1
	- trinkets 3.0.4
[17:35:26] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.4 Source=file:/C:/Users/CapoF/AppData/Roaming/gdlauncher_next/datastore/libraries/net/fabricmc/sponge-mixin/0.10.7+mixin.0.8.4/sponge-mixin-0.10.7+mixin.0.8.4.jar Service=Knot/Fabric Env=CLIENT
[17:35:26] [main/INFO]: Compatibility level set to JAVA_16
[17:35:28] [main/INFO]: Downloading deobfuscation mappings: 1.17.1+build.64 for the first launch
[17:35:32] [main/ERROR]: Mixin apply for mod servercore-fabric failed servercore.mixins.json:performance.FilledMapItemMixin -> net.minecraft.class_1806: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException @At("INVOKE") on net/minecraft/class_1806::onlyUpdateIfLoaded with priority 1000 cannot inject into net/minecraft/class_1806::method_7998(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;Lnet/minecraft/class_22;)V merged by net.id.aether.mixin.item.FilledMapMixin with priority 1000 [PREINJECT Applicator Phase -> servercore.mixins.json:performance.FilledMapItemMixin -> Prepare Injections ->  -> redirect$zpo000$onlyUpdateIfLoaded(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2818; -> Prepare]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: @At("INVOKE") on net/minecraft/class_1806::onlyUpdateIfLoaded with priority 1000 cannot inject into net/minecraft/class_1806::method_7998(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;Lnet/minecraft/class_22;)V merged by net.id.aether.mixin.item.FilledMapMixin with priority 1000 [PREINJECT Applicator Phase -> servercore.mixins.json:performance.FilledMapItemMixin -> Prepare Injections ->  -> redirect$zpo000$onlyUpdateIfLoaded(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2818; -> Prepare]
	at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.64) ~[?:?]
	at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:229) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:136) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.8.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.method_10234(Registry:176) ~[?:?]
	at net.minecraft.util.registry.Registry.method_34031(Registry:275) ~[?:?]
	at java.util.LinkedHashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.<clinit>(Registry:274) ~[?:?]
	at net.minecraft.Bootstrap.initialize(Bootstrap:44) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:139) [intermediary-1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.8.jar:?]
[17:35:32] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.ExceptionInInitializerError
	at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.64) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:601) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77) [fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.12.8.jar:?]
Caused by: java.lang.ExceptionInInitializerError
	at net.minecraft.Bootstrap.initialize(Bootstrap:44) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:139) ~[1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) ~[fabric-loader-0.12.8.jar:?]
	... 2 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1806 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:234) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:136) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.8.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.method_10234(Registry:176) ~[?:?]
	at net.minecraft.util.registry.Registry.method_34031(Registry:275) ~[?:?]
	at java.util.LinkedHashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.<clinit>(Registry:274) ~[?:?]
	at net.minecraft.Bootstrap.initialize(Bootstrap:44) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:139) ~[1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) ~[fabric-loader-0.12.8.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:229) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:136) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.8.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.method_10234(Registry:176) ~[?:?]
	at net.minecraft.util.registry.Registry.method_34031(Registry:275) ~[?:?]
	at java.util.LinkedHashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.<clinit>(Registry:274) ~[?:?]
	at net.minecraft.Bootstrap.initialize(Bootstrap:44) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:139) ~[1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) ~[fabric-loader-0.12.8.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [servercore.mixins.json:performance.FilledMapItemMixin] from phase [DEFAULT] in config [servercore.mixins.json] from mod [servercore-fabric] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:229) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:136) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.8.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.method_10234(Registry:176) ~[?:?]
	at net.minecraft.util.registry.Registry.method_34031(Registry:275) ~[?:?]
	at java.util.LinkedHashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.<clinit>(Registry:274) ~[?:?]
	at net.minecraft.Bootstrap.initialize(Bootstrap:44) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:139) ~[1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) ~[fabric-loader-0.12.8.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: @At("INVOKE") on net/minecraft/class_1806::onlyUpdateIfLoaded with priority 1000 cannot inject into net/minecraft/class_1806::method_7998(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;Lnet/minecraft/class_22;)V merged by net.id.aether.mixin.item.FilledMapMixin with priority 1000 [PREINJECT Applicator Phase -> servercore.mixins.json:performance.FilledMapItemMixin -> Prepare Injections ->  -> redirect$zpo000$onlyUpdateIfLoaded(Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2818; -> Prepare]
	at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.64) ~[?:?]
	at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1043) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:393) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.10.7+mixin.0.8.4.jar:0.10.7+mixin.0.8.4]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:229) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:136) ~[fabric-loader-0.12.8.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155) ~[fabric-loader-0.12.8.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.method_10234(Registry:176) ~[?:?]
	at net.minecraft.util.registry.Registry.method_34031(Registry:275) ~[?:?]
	at java.util.LinkedHashMap.forEach(Unknown Source) ~[?:?]
	at net.minecraft.util.registry.Registry.<clinit>(Registry:274) ~[?:?]
	at net.minecraft.Bootstrap.initialize(Bootstrap:44) ~[?:?]
	at net.minecraft.client.main.Main.main(Main:139) ~[1.17.1.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:599) ~[fabric-loader-0.12.8.jar:?]
	... 2 more
```

### Other mods

```shell
Architectury-2.6.35
cloth-api-2.0.54
cloth-config-5.1.40
fabric-api-0.43.1
notenoughcrashes-4.1.2
servercore-1.2.5
```

**Comment by CapoFantasma97** [comment]
Wesley1808/ServerCore-Fabric#6

**Comment by itsdinkd** [comment]
That mod is incompatible with a lot of other mods. I wouldn't use it.

**Comment by Wesley1808** [comment]
> That mod is incompatible with a lot of other mods. I wouldn't use it.

I checked out some popular 1.18 modpacks with a few hundred mods with it, and it worked perfectly fine.
Ofcourse, I can't account for individually testing each and every modded entity and item and its features, but it seems to run pretty much all areas that SC actually modifies without issues.
If you happen to know of any others that might still break, reporting them is always an option so I can fix them ;)

For The Aether Reborn specifically, I could optionally disable the mixin on my end if this mod is loaded (for  1.18+ versions).
The majority of SC's mixins are independent from one another, including the one listed here.

**Comment by SollyW** [comment]
Blame azzy

**Comment by murkje** [comment]
> > That mod is incompatible with a lot of other mods. I wouldn't use it.
> 
> I checked out some popular 1.18 modpacks with a few hundred mods with it, and it worked perfectly fine. Ofcourse, I can't account for individually testing each and every modded entity and item and its features, but it seems to run pretty much all areas that SC actually modifies without issues. If you happen to know of any others that might still break, reporting them is always an option so I can fix them ;)
> 
> For The Aether Reborn specifically, I could optionally disable the mixin on my end if this mod is loaded (for 1.18+ versions). The majority of SC's mixins are independent from one another, including the one listed here.

How would you disable the mixin for this? Asking for a friend.

**Comment by Wesley1808** [comment]
By using a mixin config plugin. It allows you to stop certain mixins from loading if a certain requirement is met.

**Comment by Jack-Papel** [comment]
This issue has been resolved in dev builds, and will be in our next release. We have a release candidate coming tonight and a full release coming on the 14th. Thanks for reporting!

**Comment by Jack-Papel** [comment]
This has been fixed in 1.6.5+. Thanks for reporting!
